examples/treeview/treeview-navigation.html

```diff
@@ -0,0 +1,741 @@
+          <tbody>
+            <tr data-test-id="tree-role">
+              <th scope="row"><code>tree</code></th>
+              <td></td>
+              <td><code>ul</code></td>
+              <td>
+                <ul>
+                  <li>Identifies the <code>ul</code> element as a <code>tree</code> widget.</li>
+                  <li>
+                    Because focus movement in the tree is managed with a
+                    <a href="../../#kbd_roving_tabindex">roving tabindex</a>,
+                    the <code>tree</code> container does not need a <code>tabindex</code> attribute.
+                  </li>
+                </ul>
+              </td>
+            </tr>
+            <tr data-test-id="tree-aria-label">
+              <td></td>
+              <th scope="row"><code>aria-label="Mythical University"</code></th>
+              <td><code>ul</code></td>
+              <td>Provides an accessible name for the <code>tree</code>.</td>
+            </tr>
+            <tr data-test-id="treeitem-role">
+              <th scope="row"><code>treeitem</code></th>
+              <td></td>
+              <td><code>a</code></td>
+              <td>Identifies the  element as a <code>treeitem</code>.</td>
+            </tr>
+            <tr data-test-id="treeitem-tabindex">
+              <td></td>
+              <th scope="row"><code>tabindex="-1"</code></th>
+              <td><code>a</code></td>
+              <td>
+                <ul>
+                  <li>Makes the element with the <code>treeitem</code> role focusable without including it in the tab sequence of the page.</li>
+                  <li>All <code>treeitem</code> elements are focusable, but only one is included in the tab sequence.</li>
+                </ul>
+              </td>
+            </tr>
+            <tr data-test-id="treeitem-tabindex">
+              <td></td>
+              <th scope="row"><code>tabindex="0"</code></th>
+              <td><code>a</code></td>
+              <td>
+                <ul>
+                  <li>Includes the element with the <code>treeitem</code> role in the tab sequence.</li>
+                  <li>Only one <code>treeitem</code> in the <code>tree</code> has <code>tabindex="0"</code>.</li>
+                  <li>In this implementation <code>tabindex="0"</code> is always on the treeitem with <code>aria-current="page"</code>.
+                  </li>
+                </ul>
+              </td>
+            </tr>
+            <tr data-test-id="treeitem-aria-current">
+              <td></td>
+              <th scope="row"><code>aria-current="page"</code></th>
+              <td><code>a</code></td>
+              <td>
+                <ul>
+                  <li>Applied to the <code>treeitem</code> that is the link to the currently displayed page.</li>
+                  <li>The visual indication of the <code>treeitem</code> with <code>aria-current</code> is an SVG circle with a transparent fill after the label text.</li>
```

Comment:
We changed this look, right?

Reply:
@mcking65 
yes. You are correct. We removed the svg circel from indicating "aria-current"

Reply:
@a11ydoer 
But we have something else in its place, right? How would you describe the appearance of the new current page indicator? And, is it made with SVG?

Reply:
I see that documentation was already updated for aria-current page with below, which is correct. 

"The visual indication of the treeitem with aria-current is a vertical bar to the left of the treeitem label."